roles/update/templates/workload_launch.sh.j2

```diff
@@ -188,6 +189,8 @@ function sanity_teardown {
     local elapsed_seconds=0
     {% if workload_sriov|default(false) | bool -%}
     openstack port delete "${SRIOV_PORT}"
+    {% elif workload_dpdk|default(false) | bool -%}
+    openstack port delete "${DPDK_PORT}"
     {% else -%}
```

Comment:
Oki, so I'm not an expert in `dpdk` - or `sriov` for that matter - but just to make sure.  Here (at L193) we don't remove the FIP from the INSTANCE.

But on L50 of that same file (`set_vm_ip` function called on L438), we do create a FIP.  So it means that just deleting the port remove the FIP.  Is that correct ?  Also, also around line 50, for `sriov `the command to get FIP was different, in the sense that creating the port seems to create the FIP.  Here for `dpdk` we don't have any special treatment. 

Overall, that looks strange to me: we only need to delete the port to get rid of the FIP, but we do have to run `openstack floating ip create` to create it.

So is that ok and it just shows my lack of knowdegde in dpdk or is there something wrong there ?

Thanks,